.github/workflows/dispatch-apps.yml

```
@@ -27,6 +27,11 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref_name }}-${{ inputs.environment }}
 
 jobs:
+
+  get-currently-released-version:
+    name: Get currently released version
+    uses: ./.github/workflows/workflow-get-currently-released-version.yml
```

actionlint

could not read reusable workflow file for "./.github/workflows/workflow-get-currently-released-version.yml": open /cache/16/5149c710-f89f-4a8a-bbc4-e7120d25035d/home/jailuser/git/.github/workflows/workflow-get-currently-released-version.yml: no such file or directory

(workflow-call)